python/tests/test_dataset.py

```diff
@@ -35,7 +35,7 @@ def test_create():
     assert sc.is_equal(d.coords['y'], y)
     assert sc.is_equal(d['xy'].data, xy)
     assert sc.is_equal(d['x'].data, x)
-    assert bool(set(d.dims) - set(['y', 'x']))
+    assert set(d.dims) == set(['y', 'x'])
```

Comment:
This is related to `.dims` not returning strings. Not sure the assertion made much sense before..